### DISCUSSION TITLE
Bookmark Specific hn comments?

### DISCUSSION THREAD

**Discussion by trySometngNew** [SimonHalvdansson/Harmonic-HN#104] [Ideas]
Hi @SimonHalvdansson,
I have an idea for feature for Harmonic app. 
Can a user bookmark Specific comment under a hn post? Would love if synced to user hn profile but if not then local Harmonic bookmark should also work. Also if implemented only locally, would be great to include it in export and import data. 

**Note**: Created new idea instead of Github Issue since this is not yet validated by you or community and ideas seem nifty discussion feature instead of issue. Let me know if you prefer issue instead.

**Comment by SimonHalvdansson** [comment]
Fixed in 3.0
